classes/class-wc-rest-connect-shipping-label-controller.php

```diff
@@ -101,21 +109,48 @@ public function update_items( $request ) {
 				$this->logger->log( $error, __CLASS__ );
 				return $error;
 			}
-			$labels_data[] = $label_data->label;
-			$labels_order_meta[] = array(
+			$label_ids[] = $label_data->label->label_id;
+
+			$label_meta = array(
 				'label_id' => $label_data->label->label_id,
 				'tracking' => $label_data->label->tracking_id,
 				'refundable_amount' => $label_data->label->refundable_amount,
 				'created' => $label_data->label->created,
 				'carrier_id' => $settings[ 'carrier' ],
 				'service_name' => $service_names[ $index ],
 			);
+
+			$package = $settings[ 'packages' ][ $index ];
+			$box_id = $package[ 'box_id' ];
+			if ( 'individual' === $box_id ) {
+				$label_meta[ 'package_name' ] = __( 'Individual packaging', 'connectforwoocommerce' );
+			} else if ( isset( $package_lookup[ $box_id ] ) ) {
+				$label_meta[ 'package_name' ] = $package_lookup[ $box_id ][ 'name' ];
+			} else {
+				$label_meta[ 'package_name' ] = __( 'Unknown package', 'connectforwoocommerce' );
```

Comment:
It's kind of an edge case, but it's possible to print a label for a box that has been removed from a packaging manager.